__tests__/utils/log.spec.js

```diff
@@ -27,41 +27,41 @@ jest.mock('kleur', () => ({
   })),
 }));
 
+const TOTAL_EXPECTED_STEPS = 6;
+
 describe('log functions', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     jest.spyOn(console, 'log').mockImplementation(() => {});
     kleur.enabled = false; // color support for unit tests is flaky, disable color for snapshots
   });
   describe('stepBanner', () => {
-    it('should output the correct string for all 5 steps', () => {
-      goToStep(1);
-      goToStep(2);
-      goToStep(3);
-      goToStep(4);
-      goToStep(5);
-      expect(console.log).toHaveBeenCalledTimes(30);
+    it(`should output the correct string for all ${TOTAL_EXPECTED_STEPS} steps`, () => {
+      for (let i = 1; i <= TOTAL_EXPECTED_STEPS; i += 1) {
+        goToStep(i);
+      }
+      expect(console.log).toHaveBeenCalledTimes(TOTAL_EXPECTED_STEPS * 6);
       // snapshot all calls all at once
       console.log.mock.calls.forEach((mockCall) => {
         expect(mockCall).toMatchSnapshot();
       });
     });
-    it('should output the correct string for all 5 steps with a banner', () => {
-      goToStep(1, 'BannerMock');
-      goToStep(2, 'BannerMock');
-      goToStep(3, 'BannerMock');
-      goToStep(4, 'BannerMock');
-      goToStep(5, 'BannerMock');
-      expect(console.log).toHaveBeenCalledTimes(35);
+
+    it(`outputs the correct string for all ${TOTAL_EXPECTED_STEPS} steps with a banner`, () => {
+      for (let i = 1; i <= TOTAL_EXPECTED_STEPS; i += 1) {
+        goToStep(i, 'BannerMock');
+      }
+      expect(console.log).toHaveBeenCalledTimes(TOTAL_EXPECTED_STEPS * 7);
       // snapshot all calls all at once
       console.log.mock.calls.forEach((mockCall) => {
         expect(mockCall).toMatchSnapshot();
```

Comment:
Creating snapshots in loops is creating too many similar snapshots to be useful. Can we do something else here? An inline snapshot may do the trick.

Reply:
i've updated to make the snapshots more readable (in my opinion)